strawberry/schema/schema.py

```diff
@@ -297,13 +297,12 @@ class Query:
             raise ValueError(f"Invalid Schema. Errors:\n\n{formatted_errors}")
 
     def get_extensions(self, sync: bool = False) -> list[SchemaExtension]:
-        extensions = []
-        if self.directives:
-            extensions = [
-                *self.extensions,
-                DirectivesExtensionSync if sync else DirectivesExtension,
-            ]
+        extensions: list[SchemaExtension] = []
```

Comment:
```suggestion
        extensions: list[type[SchemaExtension] | SchemaExtension] = []
```

I should have done this locally :D